src/js/components/utils/detailPageLayout.tsx

```diff
@@ -89,6 +88,14 @@ const DetailPageLayout = ({
                     </Link>
                   );
                 }
+                if (crumb.tourPopover) {
+                  return (
+                    <div className="slds-p-horizontal_x-small" key={idx}>
+                      {crumb.name}
+                      {crumb.tourPopover}
+                    </div>
+                  );
+                }
```

Comment:
I think it would work to just always include `{crumb.tourPopover}` and remove this conditional; if it's `undefined` then it won't render anything anyway.

Also, not a big deal but for consistency I'd say we might as well include `{crumb.tourPopover}` in the above `<Link>` version too.